projects/igniteui-angular/src/lib/date-picker/date-picker.component.ts

```diff
@@ -563,8 +571,22 @@ export class IgxDatePickerComponent implements IDatePicker, ControlValueAccessor
      *<igx-date-picker (onClose)="close($event)" cancelButtonLabel="cancel" todayButtonLabel="today"></igx-date-picker>
      *```
      */
+    @DeprecateProperty(`'onClose' @Ouput property is deprecated\nUse 'onClosed' instead.`)
```

Comment:
Typo `@Ouput`. Replace `\n` with `.` so there won't be a new line in the output.